hr.rst

```diff
@@ -0,0 +1,10 @@
+:banner: banners/project.jpg
```

Comment:
why image name is project? its supposed to be about HR. and usually its png?

Reply:
All banners seems to be in jpg, if you look at the image you will see that it fits the purpose, however its true i took an existing banner, will look for a new one.